contracts/erc721/UserMintableERC721.sol

```diff
@@ -33,6 +34,7 @@ contract UserMintableERC721 is Initializable, ERC721, ERC721Enumerable, ERC721Me
   // array of field names that are being proved using the document root and precise-proofs
   bytes[] internal _mandatoryFields;
 
+  string _tokenUriBase;
```

Comment:
add a comment please here what this field is about and *why* it is stored here instead of with each token.